Dis 73 25.01.00 custom form permissions

AlexanderBlanchardAC opened the pull request:
This patch limits the custom forms displayed in the dropdown when adding a custom form to a custom page to those of the user's home library if the user's permissions are set to Administer Library Custom Forms. See test plan on JIRA.

https://aspen-discovery.atlassian.net/browse/DIS-73

mdnoble73 commented:
This needs to take into account any additional libraries/locations that the user is allowed to administer.

AlexanderBlanchardAC commented:
> This needs to take into account any additional libraries/locations that the user is allowed to administer.

It should! I have added a commit to address this and updated the test plan on Jira.
